lib/bolt/core/src/dep/terraform/servers.rs

```diff
@@ -31,6 +31,14 @@ pub fn build_servers(
 ) -> Result<HashMap<String, Server>> {
 	let ns = ctx.ns_id();
 
+	// HACK: Linode requires tags to be > 3 characters. We extend the namespace to make sure it
+	// meets the minimum length requirement.
+	let ns_long = if ns.len() < 3 {
+		format!("rivet-{ns}")
+	} else {
+		ns.to_string()
+	};
+
```

Comment:
Could this make some tags inconsistent? Like some might be `123` while others would be `rivet-12`. I wonder if it might make sense to just have this be the format for everything. Though I'm not too sure what `ns` could contain, so this might not make sense.

Reply:
Yeah, that makes more sense. Server tags aren't actually used anywhere since Terraform tracks the actual server IDs internally. It's just for the rare case you need to go hit the server provider directly.

3488fbf7